CMakeLists.txt

```diff
@@ -265,7 +265,7 @@ set(ADIOS2_CONFIG_OPTS
     DataMan DataSpaces HDF5 HDF5_VOL MHS SST Fortran MPI Python PIP Blosc2 BZip2
     LIBPRESSIO MGARD MGARD_MDR PNG SZ ZFP DAOS IME O_DIRECT Sodium Catalyst SysVShMem UCX
     ZeroMQ Profiling Endian_Reverse Derived_Variable AWSSDK XRootD GPU_Support CUDA Kokkos
-    Kokkos_CUDA Kokkos_HIP Kokkos_SYCL Campaign
+    Kokkos_CUDA Kokkos_HIP Kokkos_SYCL Campaign KVCACHE
```

Comment:
need to add as at adios2_option in this file.

Reply:
added